spread-shellcheck

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+SHELLCHECK_SHELL=${SHELLCHECK_SHELL:-bash}
+NO_FAIL=${NO_FAIL:-}
+D=${D:-}
+V=${V:-}
+# file with list of test file that must successfully validate, one file per line
```

Comment:
tests/unit/spread-shellcheck/must:1:25: "successfuly" is a misspelling of "successfully"